### DISCUSSION TITLE
Multiple events save to Google Calendar

### DISCUSSION THREAD

**Discussion by belkocik** [add2cal/add-to-calendar-button#136] [Q&A] [answered]
Hello, is it possible to save multiple events to Google Calendar with one button with this library or in other way?
Got an erray with objects:
![image](https://user-images.githubusercontent.com/62507500/185623462-a60a9b7d-2315-4898-a2f9-8aa5e97a0ab4.png)
with params looking like this
```js
type MixedRubbish = {
  id: number;
  created_at: Date;
  name: string;
  description: string;
  startDate: Date;
  endDate: Date;
  collectRubishDate: string;
};
```

**Comment by jekuer** [comment]
No yet, but working on it :) .
Will keep you posted.

**Comment by mredodos** [comment]
+1

**Comment by jekuer** [comment, marked as answer]
Implemented with new v1.18! 🥳

For the Google case, it works in 2 ways:
- if you use the iCal option on an Android device, the Google Cal app will open and offer you to import the multiple events.
- using the Google option (no matter the device), it will show you the individual events in a pop-up and you need to add them one by one. Unfortunately, there is no better way at the moment. We support the user by marking already added events.
